backend/cmd/api/occurrences_integration_test.go

```diff
@@ -46,8 +46,9 @@ func testCount(t *testing.T, data string, body string, expected int) {
 func testAggregation(t *testing.T, data string, body string, expected string) {
 	testutils.ParallelTestWithDb(t, data, func(t *testing.T, db *gorm.DB) {
 		repo := repository.NewGermlineSNVOccurrencesRepository(db)
+		facetsRepo := repository.NewFacetsRepository()
```

Comment:
Could you add an integration test for this new use case ?

Reply:
Added :heavy_check_mark: